rsts/deployment/gcp/manual.rst

```diff
@@ -578,12 +579,12 @@ Accessing Flyte Console (web UI)
 * Ignore the certificate error if using a self-signed cert
```

Comment:
Please check: https://<FLYTE-ENDPOINT>/console
It is not appearing as a complete link.

Reply:
I think it's ok, User need to replace his <FLYTE-ENDPOINT> first